README.md

```diff
@@ -245,13 +245,13 @@ These variables are the defaults of our roles, if you want to override the prope
 | access_token_secret             | The secret used for access tokens JWT signature                                                                  | Generated by playbook                                 |
 | refresh_token_secret            | The secret used for refresh tokens JWT signature                                                                 | Generated by playbook                                 |
 | web_admin_username              | Username of the admin user in web application                                                                    | admin                                                 |
-| enable_alerting                 | Enable the alerting mechanism on web project                                                                     | false                                                 |
-| alert_sender                    | Email address used as the "from" address in alerts                                                               |                                                       |
-| alert_recipient                 | Email address to receive alert notifications                                                                     |                                                       |
-| smtp_server                     | IP address of the SMTP server                                                                                    |                                                       |
-| smtp_port                       | Port number of SMTP server                                                                                       |                                                       |
-| smtp_user                       | Username for SMTP authentication                                                                                 |                                                       |
-| smtp_password                   | Password for SMTP authentication                                                                                 |                                                       |
+| enable_alerting                 | Enable the alerting mechanism on web project                                                                     | null                                                  |
```

Comment:
It's usually a bad idea to have nullable booleans, hence making them a ternary value. Most of the time, you are overloading the variable with an implicit meaning (as in this case).

 Have you considered alternatives? Like, for example, another flag specific for the purpose.